UnleashedRecomp/main.cpp

```diff
@@ -61,8 +61,8 @@ void KiSystemStartup()
 {
     const auto gameContent = XamMakeContent(XCONTENTTYPE_RESERVED, "Game");
     const auto updateContent = XamMakeContent(XCONTENTTYPE_RESERVED, "Update");
-    XamRegisterContent(gameContent, GAME_INSTALL_DIRECTORY "/game");
-    XamRegisterContent(updateContent, GAME_INSTALL_DIRECTORY "/update");
+    XamRegisterContent(gameContent, (g_gameInstallPath / "game").string());
+    XamRegisterContent(updateContent, (g_gameInstallPath / "update").string());
```

Comment:
The string() method can fail with an exception if a Unicode character is present. While the current path won't have it, we should leave the code ready by using the following pattern instead.
```
std::u8string gamePathU8 = (g_gameInstallPath / "game").u8string();
XamRegisterContent(gameContent, (const char*)(gamePathU8.c_str()));
```
Which as intended, will pass along an std::string with UTF-8 characters, which is what the rest of the code expects.

Reply:
I applied your suggestion. Thanks for pointing that out!